ddp-workspace/projects/ddp-sdk/src/lib/components/validationMessage.component.ts

```diff
@@ -13,8 +13,14 @@ import * as _ from 'underscore';
         <ng-template #singleMessage>
             <span [innerHTML]="isArrayOfMessages() ? message[0] : message" routeTransformer></span>
         </ng-template>
-    </div>`
+    </div>`,
+    changeDetection: ChangeDetectionStrategy.OnPush
 })
+/**
+ * Display a validation message.
+ * Note we are using OnPush detection strategy. Make sure when modifying the message that you are creating
+ * passing new object, not just modifying the message object. Important when passing arrays of messages.
+ */
 export class ValidationMessage {
   @Input() public message: string[] | string | null;
 
```

Comment:
Was sometimes seeing some delays displaying and hiding validation messages. Changing to OnPush might help and should not break anything given the one @Input() variable.